release/models/interfaces/openconfig-interfaces.yang

```diff
@@ -720,21 +727,26 @@ module openconfig-interfaces {
       type boolean;
       description
         "When set to true, the interface is a dedicated
-        management interface that is not connected to dataplane
-        interfaces.  It may be used to connect the system to an
-        out-of-band management network, for example.";
+        management interface connected to the CPU and available
```

Comment:
I think `connected to the CPU` phrasing is imprecise. They're co-located on the same board, but the CPU (a chip) is not a direct owner of the interface and does not have a direct connection. If we start counting indirect connections, then any regular interface is connected to the CPU.

Reply:
Perhaps all we can say that  is unique about a management connected interface is:

1. it is not connected to an INTEGRATED_CIRCUIT.
2. It is externally accessible to the device for user management access.

Is this more clear and precise enough to disambiguate from a 'cpu' interface?

Reply:
That seems reasonable. 
FWIW I'm happy with the current definition of `management` leaf. In my view, ambiguity was related mostly to the definition `cpu` leaf (and the way it is used in gnmi-1.22 https://github.com/openconfig/featureprofiles/blob/4cff2c8fa1815887a481b0b051fbf077464c47ad/feature/platform/controllercard/tests/port/README.md?plain=1#L32-L33)

Reply:
I fixed the description for gnmi-1.22 here: https://github.com/openconfig/featureprofiles/pull/3364

I also removed the "connected to the CPU" part of the description.